app/components/Search/Search.css

```diff
@@ -73,28 +71,26 @@
   display: grid;
   grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
   list-style-type: none;
+  gap: var(--spacing-xs);
 }
 
 .quickLinkWrapper {
   padding: 0;
 }
 
 .quickLink {
-  padding: 4px 15px;
+  padding: var(--spacing-sm) var(--spacing-md);
   box-sizing: border-box;
   font-size: var(--font-size-lg);
   border-radius: var(--border-radius-md);
   transition: background-color var(--linear-fast);
+  color: var(--lego-font-color);
```

Comment:
<img width="1385" alt="image" src="https://github.com/webkom/lego-webapp/assets/69514187/cd4473b8-32f9-4c7a-b681-1e437b16b274">

Unrelated to the spacing, but about time!

Reply:
Looks a lot cleaner <33